data_steward/tools/import_rdr_omop.sh

```diff
@@ -8,7 +8,6 @@ set -ex
 USAGE="tools/import_rdr_omop.sh
     --rdr_project <PROJECT where rdr files are dropped>
     --rdr_directory <DIRECTORY, not including the gs:// bucket name>
-    --output_dataset <DATA SET>
```

Comment:
It seems like this should have become `dataset_release_tag`.  Lines 61-63 are still using the old convention of the day the import was run for naming the dataset instead of the `{YYYY}q{X1}r{X2}` format in the naming convention document.

Reply:
this script will be used to load the monthly RDR dumps to BigQuery. we snapshot this dataset and apply the cleaning rules while we start running the pipeline using `create_ehr_and_rdr_snapshot.sh` the first step of running the cdr pipeline

Reply:
I think I understand.  It sounds like we are loading RDR data into datasets on a monthly basis, but that is not where the pipeline starts.  
When this runs lines 61 -63, it will create a BigQuery dataset with a name formatted as:  `YYYMMDD_rdr`, `YYYMMDD_rdr_backup`, and `YYYMMDD_rdr_staging`.   Our naming conventions document says they will be named using the convention `{YYYY}q{X1}r{X2}_rdr` and `{YYYY}q{X1}r{X2}_rdr_backup`.  

The staging environment isn't even indicated for the rdr dataset.  Either the code is wrong or the documentation needs updating or both??

Reply:
The dataset naming convention document is for the naming convention that we will be using for the datasets that are being created as part of the pipeline. 
So my understanding from that is - we will load an rdr dump into bigquery and name it as rdr{YYMMDD}. which will be the raw data that we get from the rdr and when we run the CDR generation pipeline we will be snapshotting the dataset which we decide on using and apply the cleaning rules on it which will use the CDR naming convention which is {YYYY}q{X1}r{X2}_rdr.